### PR TITLE
AWS Lambda supported protocols fixes

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/DOCS-1362 remove grpc as supported protocol. See link to slack convo in docs ticket.

Add http supported protocol to first plugin version. Fix file name so 3.0.x does not 404.

Direct review links:

- https://deploy-preview-2394--kongdocs.netlify.app/hub/kong-inc/aws-lambda/0.1-x.html#configuration
- https://deploy-preview-2394--kongdocs.netlify.app/hub/kong-inc/aws-lambda/1.0-x.html#configuration
- https://deploy-preview-2394--kongdocs.netlify.app/hub/kong-inc/aws-lambda/3.0.x.html#configuration
https://deploy-preview-2394--kongdocs.netlify.app/hub/kong-inc/aws-lambda/#configuration

### CODE REVIEW

**Review by GayleNeumann — APPROVED** [review]
LGTM

File: app/_hub/kong-inc/aws-lambda/1.0-x.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: AWS Lambda
 publisher: Kong Inc.
-version: 1.0.x
+version: 1.0-x
```
Comment (Tieske) [inline app/_hub/kong-inc/aws-lambda/1.0-x.md:4]: why the "-" here?

Comment (tlblessing) [reply app/_hub/kong-inc/aws-lambda/1.0-x.md:4]: > why the "-" here?

@Tieske I was just matching the internal version with the file name and versions drop down menu. If these are wrong, that will have to be another PR.

image

Comment (Tieske) [reply app/_hub/kong-inc/aws-lambda/1.0-x.md:4]: afaik only the Kong Enterprise packages were ever versioned with a -, eg. 0.34-1, etc. But that was dropped when the version got synced with CE. No plugin was ever versioned like that

**Pull request merged**